__tests__/app.test.js

```diff
@@ -54,6 +54,49 @@ describe("/api/articles", () => {
   });
 });
 
+describe("/api/articles/:article_id", () => {
+  test("GET: 200 - Responds with single article with the correct article_id", () => {
+    return request(app)
+      .get("/api/articles/2")
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article.article_id).toBe(2);
+      });
+  });
+  test("GET: 200 - The response article to have the correct structure / properties", () => {
+    return request(app)
+      .get("/api/articles/3")
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article.article_id).toBe(3);
+        expect(article).toMatchObject({
+          author: expect.any(String),
+          title: expect.any(String),
+          article_id: expect.any(Number),
```

Comment:
You're already asserting the value of article_id above, so feel free to remove or replace the expect statement before.